docs/myrocks/variables.md

```diff
@@ -2582,6 +2754,22 @@ Specifies whether MyRocks should use `fsync` instead of `fdatasync`
 when requesting a sync of a data file.
 Disabled by default.
 
+### `rocksdb_use_write_buffer_manager`
+
+| Option       | Description         |
+|--------------|---------------------|
+| Command-line | --rocksdb_use_write_buffer_manager |
+| Dynamic      | No                   |
+| Scope        | Global               |
+| Data type    | Boolean              |
+| Default      | OFF                  |
+
+The variable was implemented in [Percona Server for MySQL 8.0.33-25](../release-notes/Percona-Server-8.0.33-25.md).
+
+If enabled, allows using write buffer manager. 
```

Comment:
`Allows to turn on the write buffer manager (WriteBufferManager) from cnf files.`
`This option is related to rocksdb_charge_memory.`
`This is an experimental feature that may be removed in the future.`

Reply:
Updated